plugins/janus_videoroom.c

```diff
@@ -4369,6 +4374,34 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	janus_mutex_unlock(&sessions_mutex);
 }
 
+static void janus_videoroom_hangup_subscriber(janus_videoroom_subscriber * s) {
+	/* already hangup */
```

Comment:
Just a nit, but can you please make sure all comments start with a capital letter?